proposals/README.md

```diff
@@ -0,0 +1,56 @@
+# Proposals
+
+This proposal process is intended for significantly impactfull additions to the Parquet spec. The goal is to facilitate those projects and help them being contributed to Parquet.
+For example, changes that are not backward compatible like adding a new encoding or a new compression algorithm (older implementations can not read new files).
```

Comment:
Nit, we refer to this elsewhere as forward compatible: https://github.com/apache/parquet-format/blob/master/CONTRIBUTING.md#general-guidelinespreferences-on-additions